Expand textareas on page load

BartChris opened the pull request:
This PR tries to make a quality of life improvement for disabled textareas. Those textareas might contain text which spans over multiple lines. If the textarea has editing disabled, the box cannot be expanded and the editor does not see the full content. I am using Javascript to expand the textareas to the necessary size.

With this change the editor always sees the full content.

![image](https://github.com/user-attachments/assets/7d6faf8e-e3f2-4ad3-9d18-cd47098c30fd)

oliver-stoehr reviewed (commented):
The script works fine and displays the textAreas with full height.
Is resetting the height really necessary? I also tested the script without `textarea.style.height = "auto"; // Reset height first` which worked in all browsers.
Is there a special scenario where this line is necessary?

BartChris commented:
I removed that line, which might be necessary, if the box needs to be shrinked if an initial size is larger than the actual content. I think we do not have this scenario and therefor removed it. 

I added another check to ensure the listener is only registered once when having multiple textareas on the screen.

solth commented:
@BartChris perhaps it would make sense to backport this QoL improvement for Kitodo.Production 3.8 as well?
